src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -32,6 +33,23 @@ export interface BreadcrumbProps {
   id?: string;
 }
 
+const breadcrumbTextLength = 60;
+
+// Truncate breadcrumb text if beyond 60 characters then add ellipsis at the end.
+const breadcrumbText = (text) => {
+  
+  if (text.length <= breadcrumbTextLength) {
+    return text;
+  }
+
+  const subString = text.substr(0, breadcrumbTextLength - 1);
+  return (
+    <Tooltip content={text} id={`breadcrumb-${text}-tooltip`}>
```

Comment:
To add a better id instead of `text`, you can try to shorten it and remove the the white space(s). You also have other better variables to use, such as `breadcrumbsID` and `index` that is available to use when calling `breadcrumbText`.